systemtests/singlehost/sanity_test.go

```diff
@@ -194,3 +195,72 @@ func TestSingleHostMultiVlanPingFailure_sanity(t *testing.T) {
 		utils.DockerCleanup(t, node, "myContainer2")
 	}()
 }
+
+// Default Network Assignment and freeing
+func TestSingleHostDefaultNetwork_sanity(t *testing.T) {
```

Comment:
It might be good to test the following:

- verify the `ip route` shows the default gateway as the one corresponding to `orange` network.
- And that when default-network is removed that `ip route` no longer has default gateway set.